client/components/recycleAndReuseComponents/VerifyItem.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { 
+    Alert, 
+    AlertIcon, 
+    Box, 
+    Button, 
+    Checkbox, 
+    CloseButton, 
+    Flex, 
+    HStack, 
+    ListItem, 
+    Select, 
+    Spacer, 
+    Text, 
+    UnorderedList, 
+    VStack 
+} from '@chakra-ui/react'
+import Image from 'next/image'
+
+const VerifyItem = ({ items, setItems, navigateToTakeAction }) => {
+    // TODO: figure out how this is populated
+    const getInvalidItems = (items) => {
+        return ["Cheese", "Cheeseburger"]
+    }
```

Comment:
@neozhixuan @CippyLOL Please help regarding this one 🤔

Reply:
Can we change isbluebinrecyclable to an integer? 0 for bluebin recyclable, 1 for non-bluebin recyclable, 2 for general waste like these above. I think that would work!

Reply:
So items from the database will either be 'blue-bin recyclable', 'non-blue bin recyclable' or 'general waste'. Is that right?